components/DeleteConvoButton.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+import deleteConversation from '@/utils/messaging/deleteConversation';
+
+interface DeleteButtonProps {
+  convoId?: number;
+  title: string;
+}
+
+export default function DeleteConvoButton({
+  convoId,
```

Comment:
Could you make this uniform with the rest of the feature and call it "conversation_id" I know it's longer but it will probably help us read through and trace it later if needed.